susemanager-utils/susemanager-sls/salt/bootstrap/init.sls

```diff
@@ -76,9 +76,17 @@ no_ssh_push_key_authorized:
   #end of expections
 {%- endif %}
 
+# openEuler Family. This OS is based in RedHat, but declares a separate family
+{%- if grains['os_family'] == 'openEuler' %}
+  {% set os_base = grains['os'] %}
+  {% set osrelease_major = grains['osrelease_info'][0] %}
+  {% set osrelease_minor = '0' ~ grains['osrelease_info'][1] %}
```

Comment:
Please verify all the `os*` grains... I am quite there was one that gives the full release, including the 0?

This will break if openEuler ever has a release in October or later.

Reply:
Fixed.